pkgs/by-name/cm/cmake/setup-hook.sh

```diff
@@ -131,6 +131,9 @@ cmakeConfigurePhase() {
         enableParallelBuilding=1
         echo "cmake: enabled parallel building"
     fi
+    if [[ $enableParallelBuilding -ne 0 ]]; then
```

Comment:
Even tho not strictly necessary in `[[`, it is still good practice to always quote `$var`s. It is also done that way in the other places where `[[` is used in this file.